### PR TITLE
Adding monitoring assembly to troubleshooting guide

### PR DESCRIPTION
**Pull request by pneedle-rh**
This PR adds the troubleshooting modules that were merged in [this related PR](https://github.com/openshift/openshift-docs/pull/24722) to the Troubleshooting Guide.

Applies to master and branch/enterprise-4.6.

The preview is [here](https://troubleshooting_guide_4_6_monitoring_addition--ocpdocs.netlify.app/openshift-enterprise/latest/support/troubleshooting/investigating-monitoring-issues.html).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please.

File: support/troubleshooting/investigating-monitoring-issues.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[id="investigating-monitoring-issues"]
+= Investigating monitoring issues
+include::modules/common-attributes.adoc[]
+:context: investigating-monitoring-issues
+
+toc::[]
+
```
Comment (kalexand-rh) [inline support/troubleshooting/investigating-monitoring-issues.adoc:7]: You need an intro sentence.

**Review by kalexand-rh — COMMENTED** [review]
One small thing, then it LGTM!

File: support/troubleshooting/investigating-monitoring-issues.adoc (hunk 1 above)
Comment (pneedle-rh) [reply support/troubleshooting/investigating-monitoring-issues.adoc:7]: @kalexand-rh thank you for your review. Can you please take a look at the intro that I have now added? Thanks!

Comment (pneedle-rh) [reply support/troubleshooting/investigating-monitoring-issues.adoc:7]: One point to note is that the second paragraph in the intro would need to be updated when we add further modules to this page. But, at present I don't feel that there are enough modules in this assembly to say, "You can follow these procedures if you experience issues with the monitoring stack".

File: support/troubleshooting/investigating-monitoring-issues.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides monitoring for core platform components. In {product-title} {product-version}, cluster administrators can optionally enable monitoring for user-defined projects.
+
+You can follow these procedures if your own metrics are unavailable, or if Prometheus is consuming a lot of disk space.
```
Comment (kalexand-rh) [inline support/troubleshooting/investigating-monitoring-issues.adoc:10]:
```suggestion
You can follow these procedures if your own metrics are unavailable or if Prometheus is consuming a lot of disk space.
```

File: support/troubleshooting/investigating-monitoring-issues.adoc (hunk 1 above)
Comment (kalexand-rh) [reply support/troubleshooting/investigating-monitoring-issues.adoc:7]: I think that's fine! I suggest adding a comment to remind whoever might add the next module to this assembly to also update that sentence.

File: support/troubleshooting/investigating-monitoring-issues.adoc (hunk 2 above)
Comment (pneedle-rh) [reply support/troubleshooting/investigating-monitoring-issues.adoc:10]: Resolved.

File: support/troubleshooting/investigating-monitoring-issues.adoc (hunk 1 above)
Comment (pneedle-rh) [reply support/troubleshooting/investigating-monitoring-issues.adoc:7]: That's a great idea, thank you. Resolved.

**Pull request merged**